projects/plugins/jetpack/class.jetpack-plan.php

```diff
@@ -48,6 +48,7 @@ class Jetpack_Plan {
 				'social-previews',
 				'videopress',
 				'videopress/video',
+				'v6-video-frame-poster',
```

Comment:
If you want the extension to be available to everyone, I would recommend not adding it here.